modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1193 @@
+
+This update corrects this: `skip_focus: true` is honored, as expected, and the sidebar does not steal focus.
+
+
+==== Comments did not blocking the Save UI while saving the data in callback mode
+
+When Comments was working in callback mode, it was possible to send multiple `save` actions from the Comment text-entry UI.
+
+Save actions are asyncrhonous, so this made it possible to generate multiple saves of the same comment.
+
+This update disables the *Save* UI when a comment is being saved, ensuring only one copy of a comment is saved at a time.
+
+
+==== A selected `hr` block could be commented on when it should not have been possible
+
+A new feature of the Comments plugin is support for annotating many block-level elements directly, using the editor annotator API (see above).
+
+A bug in this new feature, however, made it possible to, at least apparently, add a comment to an `+<hr>` block.
+
+The comment did not, in fact, attach to the horizontal rule. It attached to the Comment sidebar, which resulted in the comment showing even when surrounding paragraphs were clicked in or selected.
+
+With this update, `+<hr>` blocks specifically cannot have comments added to them.
+
+
+==== Comment text was retained in the comment placeholder after more than one comment had been submitted
+
+If a second comment was added to a given element, the re-displayed ‘empty’ Comment text entry field presented with the text of the second comment in this field rather than the default placeholder text, *Add comment…*.
+
+If a third comment was added, the re-displayed Comment text entry field then presented with the text of the third comment, and so on.
+
+This update corrects this. When the Comment sidebar re-displays the Comment text entry field for a further comment, the previous comment is not retained and presented. Rather, the *Add comment…* placeholder text displays, as expected.
+
+
+==== The comment textarea will now shrink, as well as grow, in response to comment content changes
+
+Previously, the comment text entry field would grow larger to more usefully display a long comment-in-progress but it did not shink back to its orignal size when the text in this field was deleted.
+
+This Comments update corrects this. When text is deleted from a grown-larger Comments text-entry field, this field now snaps back to its original size.
+
+
+
+For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Comments plugin].
+
+
+
+
+=== Enhanced Media Embed 3.0.1
+
+The {productname} 6.1 release includes an accompanying release of the **Enhanced Media Embed** premium plugin.
+
+**Enhanced Media Embed** 3.0.1 fixes the following bug:
+
+==== Console warnings were logged incorrectly when the media plugin was not used
+
+Previously, a ‘`media_url_resolver` not registered’ warning was printed to the console when the Enhanced Media Embed was not used.
```

Comment:
```suggestion
Previously, a `'media_url_resolver' not registered` warning was printed to the console when the Enhanced Media Embed was not used.
```
The markup for this didn't look right: 
![image](https://user-images.githubusercontent.com/1575550/178645365-f15d7479-f872-4208-a898-0e2bc2f2274b.png)